.pullapprove.yml

```diff
@@ -1,8 +1,10 @@
 approve_by_comment: true
-approve_regex: ^LGTM
+approve_regex: '^(lgtm|LGTM)'
```

Comment:
ALL CAPS signifies confidence in your decision. We should make sure we have confident maintainers. :P